models/user.js

```diff
@@ -42,7 +42,7 @@ const userSchema = new mongoose.Schema({
         required: [true, 'Please provide a password'],
         minlength: [8, 'Please provide a password with minimum length of 8'],
         // Make the password never show up for any get request
-        select: false
+        //select: false
```

Comment:
Please return this back to default. In a production environment when querying user credentials password should not be returned either encrypted or not